ADR/index.md

```diff
@@ -0,0 +1,48 @@
+# Architecture Decision Records
```

Comment:
We currently get an index automatically generated for this by the script: https://redhat-appstudio.github.io/book/ADR/index.html

We need to fix up the names and titles of the ADRs so that it looks clean, but if we do then we won't have to maintain a manual index.md file.

Reply:
The script is in https://github.com/redhat-appstudio/book/blob/main/.github/workflows/generate-adr-index.yml

Reply:
See also https://github.com/redhat-appstudio/book/pull/32